# Four Ultrasonic Range Sensors

#      --- Front Lf ----------- Front Rg ---
#      |                                   |
#      |                                   |
#      |                                   |
#      --- Rear Lf ------------ Rear Rg ----

# Front Right
sensor_msgs/Range sensor_FL

# Front Left
sensor_msgs/Range sensor_FR

# Rear Left
sensor_msgs/Range sensor_RR

# Rear Right
sensor_msgs/Range sensor_RL
